code/modules/surgery/cavity_implant.dm

```diff
@@ -19,6 +19,7 @@
 	preop_sound = 'sound/surgery/organ1.ogg'
 	success_sound = 'sound/surgery/organ2.ogg'
 	var/obj/item/item_for_cavity
+	var/list/heavy_weight_whitelist = list(/obj/item/transfer_valve = TRUE)
```

Comment:
```suggestion
	var/list/heavy_weight_whitelist = list(/obj/item/transfer_valve)
```

Reply:
this isn't a typecache, but this change would make it even less typecache. you need to apply typecacheof to it (say on init?) to account for subtypes properly

Reply:
ideally it would be a glob honestly but this is also ok

Reply:
I checked code and apparently is_type_in_typecache only works when you assoc them to TRUE?

Reply:
Should still autodoc it though